### PR TITLE
Improve the inclusivity of the language in Access_Control_Cheat_Sheet.md

### PR DESCRIPTION
**Pull request by ben-dale**
This change improves the inclusivity of the language in the Access Control cheat sheet.

Please let me know if I've made any errors - first time contributor!

### CODE REVIEW

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 1]
```diff
@@ -2,33 +2,33 @@
...
+The process of defining roles is usually based on analyzing the fundamental goals and structure of an organization and is usually linked to the security policy. For instance, in a medical organization, the different roles of users may include those such as a doctor, nurse, attendant, patients, etc. These members require different levels of access in order to perform their functions, but also the types of web transactions and their allowed context vary greatly depending on the security policy and any relevant regulations (HIPAA, Gramm-Leach-Bliley, etc.).
 
 An RBAC access control framework should provide web application security administrators with the ability to determine who can perform what actions, when, from where, in what order, and in some cases under what relational circumstances.
 
 The advantages of using this methodology are:
 
 - Roles are assigned based on organizational structure with emphasis on the organizational security policy
-- Easy to use
-- Easy to administer
+- Accessible to use
```
Comment (rbsec) [inline cheatsheets/Access_Control_Cheat_Sheet.md:30]: This wording sounds strange strange, I have never heard anything described as "accessible to use" in this context. These points are talking about simplicity, not accessibility - so I'd revert this change and the one below.

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 2]
```diff
@@ -42,19 +42,19 @@ Problems that can be encountered while using this methodology:
...
+Discretionary Access Control (DAC) is a means of restricting access to information based on the identity of users and/or membership in certain groups. Access decisions are typically based on the authorizations granted to a user based on the credentials they presented at the time of authentication (user name, password, hardware/software token, etc.). In most typical DAC models, the owner of the information or any resource can change its permissions at their discretion (thus the name).
 
 A DAC framework can provide web application security administrators with the ability to implement fine-grained access control. This model can be a basis for data-based access control implementation
 
 The advantages of using this model are:
 
-- Easy to use
-- Easy to administer
+- Accessible to use
```
Comment (rbsec) [inline cheatsheets/Access_Control_Cheat_Sheet.md:56]: See previous comment on this

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ The areas of caution while using MAC are:
 
 ## Permission Based Access Control
 
-The key concept in Permission Based Access Control is the abstraction of application actions into a set of *permissions*. A *permission* may be represented simply as a string-based name, for example, "READ". Access decisions are made by checking if the current user *has* the permission associated with the requested application action.
+The key concept in Permission Based Access Control is the abstraction of application actions into a set of *permissions*. A *permission* may be represented as a string-based name, for example, "READ". Access decisions are made by checking if the current user *has* the permission associated with the requested application action.
```
Comment (rbsec) [inline cheatsheets/Access_Control_Cheat_Sheet.md:94]: Why was "simply" removed? This is giving an example of a simple way to name your permissions, rather than things like the completely unreadable security descriptors that Windows uses that look like:

```
D:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;0x1200a9;;;BU)(A;CI;LC;;;BU)(A;CIIO;DC;;;BU) (A;OICIIO;GA;;;CO)
```

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 4]
```diff
@@ -2,33 +2,33 @@
...
 
 ### What is Access Control / Authorization
 
-Authorization is the process where requests to access a particular resource should be granted or denied. It should be noted that authorization is not equivalent to authentication - as these terms and their definitions are frequently confused. Authentication is providing and validating identity. Authorization includes the execution rules that determine what functionality and data the user (or Principal) may access, ensuring the proper allocation of access rights after authentication is successful.
+Authorization is the process where requests to access a particular resource should be granted or denied. It should be noted that authorization is not equivalent to authentication - as these terms and their definitions are frequently confused. Authentication is providing and validating identity. Authorization includes the operation rules that determine which functionality and data the user (or Principal) may access, ensuring the proper allocation of access rights after authentication is successful.
```
Comment (rbsec) [inline cheatsheets/Access_Control_Cheat_Sheet.md:9]: Is "operation rules" a common phrase? It's not a term I've come across before

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 5]
```diff
@@ -2,33 +2,33 @@
...
 
 Web applications need access controls to allow users (with varying privileges) to use the application. They also need administrators to manage the applications access control rules and the granting of permissions or entitlements to users and other entities. Various access control design methodologies are available. To choose the most appropriate one, a risk assessment needs to be performed to identify threats and vulnerabilities specific to your application, so that the proper access control methodology is appropriate for your application.
 
 ### Access Control Policy
 
 Why do we need an access control policy for web development?
 
-The intention of having an access control policy is to ensure that security requirements are described clearly to architects, designers, developers and support team, such that access control functionality is designed and implemented in a consistent manner.
+The intention of having an access control policy is to ensure that security requirements are described precisely to architects, designers, developers and support team, such that access control functionality is designed and implemented in a consistently.
```
Comment (rbsec) [inline cheatsheets/Access_Control_Cheat_Sheet.md:17]: Clarity and precision are not the same thing. A 200 page detailed specification of access control requirements is very *precise*, but it's not *clear*.

**Comment by ThunderSon** [comment]
I agree with the points discussed. It is important to keep in mind our audience, and the meaning of the sentence.
Another point would be to cross reference with the following [comment](https://github.com/OWASP/CheatSheetSeries/issues/369#issuecomment-611960358) as this sheet will be archived in the coming week.

**Comment by ben-dale** [comment]
@ThunderSon @rbsec Thank you greatly for your constructive comments, I perhaps jumped the gun a couple of times! In trying to make them more inclusive I perhaps could have made them less accessible to others, so thank you for casting your eyes over this change. 

@ThunderSon I wasn't aware that there were cheat sheets due to be archived! Sorry I missed that. I'll keep that comment handy going forward.

Thank you again, both! Let me know if there are any other changes to me made.

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 6]
```diff
@@ -2,25 +2,25 @@
...
 
 Web applications need access controls to allow users (with varying privileges) to use the application. They also need administrators to manage the applications access control rules and the granting of permissions or entitlements to users and other entities. Various access control design methodologies are available. To choose the most appropriate one, a risk assessment needs to be performed to identify threats and vulnerabilities specific to your application, so that the proper access control methodology is appropriate for your application.
 
 ### Access Control Policy
 
 Why do we need an access control policy for web development?
 
-The intention of having an access control policy is to ensure that security requirements are described clearly to architects, designers, developers and support team, such that access control functionality is designed and implemented in a consistent manner.
+The intention of having an access control policy is to ensure that security requirements are described cleary to architects, designers, developers and support team, such that access control functionality is designed and implemented in a consistently.
```
Comment (rbsec) [inline cheatsheets/Access_Control_Cheat_Sheet.md:17]: `clearly` is spelt incorrectly here.

File: cheatsheets/Access_Control_Cheat_Sheet.md [hunk 7]
```diff
@@ -2,25 +2,25 @@
...
 
 Web applications need access controls to allow users (with varying privileges) to use the application. They also need administrators to manage the applications access control rules and the granting of permissions or entitlements to users and other entities. Various access control design methodologies are available. To choose the most appropriate one, a risk assessment needs to be performed to identify threats and vulnerabilities specific to your application, so that the proper access control methodology is appropriate for your application.
 
 ### Access Control Policy
 
 Why do we need an access control policy for web development?
 
-The intention of having an access control policy is to ensure that security requirements are described clearly to architects, designers, developers and support team, such that access control functionality is designed and implemented in a consistent manner.
+The intention of having an access control policy is to ensure that security requirements are described clearly to architects, designers, developers and support team, such that access control functionality is designed and implemented in a consistently.
```
Comment (ThunderSon) [inline cheatsheets/Access_Control_Cheat_Sheet.md:17]: Kindly revert this change, as the new sentence is wrong.
